src/Database/Expression/WindowExpression.php

```diff
@@ -35,6 +36,16 @@ class WindowExpression implements ExpressionInterface, WindowInterface
      */
     protected $order;
 
+    /**
+     * @var array
+     */
+    protected $frame;
+
+    /**
+     * @var string
+     */
+    protected $exclusion;
```

Comment:
There's a `$this->$exclusion !== null` check in a method, should shouldn't the type be `string|null`?

Reply:
yup!